.env.example

```diff
@@ -1,6 +1,2 @@
-DATABASE_HOST=localhost
-DATABASE_PORT=5432
-DATABASE_NAME=postgres
-DATABASE_SECRET='{"username":"postgres","password":"postgres"}'  # pragma: allowlist secret
-SERVER_NAME=funding.communities.gov.localhost:8080
-SECRET_KEY=pretend_secret_key
```

Comment:
Don't we still want these in here?

Reply:
Do we? I'm not sure what they're for to be honest?

Reply:
If you run stuff locally like `uv run flask db migrate` I think it needs them? ASsuming you are doing `source .env`